### PR TITLE
feat(config): import device configs from Z-Wave Alliance (Part 2: cleanup values and labels)

### PR DESCRIPTION
**Pull request by blhoward2**
Push 2: Minor config file changes from the zWave Alliance Files (primarily adding zwa file numbers and min/max values. In support of #1349

### CODE REVIEW

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

**Comment by blhoward2** [comment]
@AlCalzone These should be ready to merge. They are all minor changes to current device files. Where there is a deletion/addition of a whole file it is because I renamed the file to match a corrected identifier.

Min/max changes were all verified eyes on. These are primarily changes from 255 to 1 for binary 0/1 values.

File: packages/config/config/devices/0x0086/dsb09.json [hunk 1]
```diff
@@ -36,7 +37,6 @@
 		"1": {
 			"label": "Voltage",
 			"description": "Voltage to calculate wattage from amperage.",
-			"unit": "V",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0086/dsb09.json:39]: That doesn't look correct. (same for the other removed units in this file)

File: packages/config/config/devices/0x0002/014g0210.json [hunk 2]
```diff
@@ -69,7 +69,6 @@
 		"3": {
 			"label": "PWM period",
 			"description": "Pulse-width modulation period",
-			"unit": "min",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0002/014g0210.json:72]: Is this correct?

File: packages/config/config/devices/0x001a/rf9542.json [hunk 3]
```diff
@@ -29,8 +30,7 @@
 	"paramInformation": {
 		"1": {
 			"label": "Delayed OFF time",
-			"description": "delay time in seconds",
-			"unit": "s",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x001a/rf9542.json:33]: this param should have `"unit": "seconds"`

File: packages/config/config/devices/0x001a/rf9542.json [hunk 4]
```diff
@@ -42,8 +42,7 @@
 		},
 		"2": {
 			"label": "Panic ON time",
-			"description": "time in seconds for panic mode",
-			"unit": "s",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x001a/rf9542.json:46]: this param should have `"unit": "seconds"`

File: packages/config/config/devices/0x001a/rf9542.json [hunk 5]
```diff
@@ -55,8 +54,7 @@
 		},
 		"3": {
 			"label": "Panic OFF time",
-			"description": "off time in seconds for panic mode",
-			"unit": "s",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x001a/rf9542.json:59]: this param should have `"unit": "seconds"`

File: packages/config/config/devices/0x0059/sir-321.json [hunk 6]
```diff
@@ -61,7 +62,6 @@
 		"3": {
 			"label": "Temperature reporting intervals",
 			"description": "Define the time interval for the temperature report that is sent.",
-			"unit": "sec",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0059/sir-321.json:64]: this param should have `"unit": "seconds"` (I think)

File: packages/config/config/devices/0x0060/hac01.json [hunk 7]
```diff
@@ -38,7 +39,6 @@
 		"2": {
 			"label": "Amount Of Delay",
 			"description": "Amount Of Delay (seconds) to send the Basic Set Off command.",
-			"unit": "s",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0060/hac01.json:41]: this param should have `"unit": "seconds"`

File: packages/config/config/devices/0x0115/zme_06433.json [hunk 8]
```diff
@@ -65,10 +66,9 @@
 		"2": {
 			"label": "Automatically switch off after",
 			"description": "If not zero, automatically switch off after a user defined time",
-			"unit": "Sec",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0115/zme_06433.json:68]: this param should have `"unit": "seconds"`

File: packages/config/config/devices/0x0115/zme_06433.json [hunk 9]
```diff
@@ -65,10 +66,9 @@
...
 			"valueSize": 2,
 			"minValue": 0,
-			"maxValue": 65535,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0115/zme_06433.json:71]: And this doesn't seem correct either. The param is used to set the interval. The options are what is wrong.
--> remove options, set "allowManualEntry" to true.

File: packages/config/config/devices/0x0115/zme_06433.json [hunk 10]
```diff
@@ -138,10 +138,9 @@
 		"5": {
 			"label": "Dimming speed",
 			"description": "Time to dim on button presses",
-			"unit": "10ms",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0115/zme_06433.json:141]: I'm sure this was here for a reason

File: packages/config/config/devices/0x0115/zme_06433.json [hunk 11]
```diff
@@ -138,10 +138,9 @@
...
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 30,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0115/zme_06433.json]: no

Comment (AlCalzone) [reply packages/config/config/devices/0x0115/zme_06433.json]: Get rid of the options instead

**Review by AlCalzone — CHANGES_REQUESTED** [review]
I don't think this is entirely correct. In cases where `allowManualEntry` is true, you cannot just take the min/max of the options. The user is free to choose any value between min and max in this case and the options are just "presets".

Besides from this, I've got a couple of comments

**Comment by marcus-j-davies** [comment]
@blhoward2 @AlCalzone ,

Not sure if this is too late, but https://github.com/zwave-js/node-zwave-js/pull/1473 mentions to remove `\r` and / or `\n`
is it too late to keep them? I can swop these new line characters for `<br />` at render time.

Some of the metadata's are very "busy" hence the line breaks make it easier on the eye.

Screenshot 2021-01-24 at 13 14 07

**Comment by AlCalzone** [comment]
@marcus-j-davies I don't want to remove it entirely, but make it consistent. Only \n, no \r, no \t, etc.

Hence why I proposed markdown so you could also render some formatting easily.

**Comment by marcus-j-davies** [comment]
As long as there are still `\n` I can use them to break the lines up.

I still think markdown is risky to use here, the layout is very organised, given that markdown can render tables, whole HTML elements etc etc, it could introduce artifacts that affects the non-markdown code.

I have only ever used markdown for read me documents, GitHub type stuff.

**Comment by marcus-j-davies** [comment]
@blhoward2 

This XML [2509](https://products.z-wavealliance.org/Products/2509/XML) 
Has spaces where I would expect them i.e **the following warning. CAUTION: This will factory**
And for the JSON Equivalent **the following warning.CAUTION: This will factory** 

Apologies, but wasn't sure if that is still a working progress, or is a glitch in the conversion routine?

File: packages/config/config/devices/0x0086/dsb09.json (hunk 1 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0086/dsb09.json:39]: This is being done by your normalize routine, so I assumed you wanted it.

File: packages/config/config/devices/0x0115/zme_06433.json (hunk 11 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0115/zme_06433.json]: These maxes come from the file. Are you sure it can handle 255?

**Comment by blhoward2** [comment]
> @blhoward2
> 
> This XML [2509](https://products.z-wavealliance.org/Products/2509/XML)
> Has spaces where I would expect them i.e **the following warning. CAUTION: This will factory**
> And for the JSON Equivalent **the following warning.CAUTION: This will factory**
> 
> Apologies, but wasn't sure if that is still a working progress, or is a glitch in the conversion routine?

I messed up at one point removing \n instead of replacing them with a space. I’ve had to chase them manually in that pile but I’ll take a look. It’s fixed in the others.

File: packages/config/config/devices/0x0115/zme_06433.json (hunk 11 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x0115/zme_06433.json]: I don't see any config parameters in the file https://products.z-wavealliance.org/products/19/json
Are you sure you didn't infer that from the option values?

**Comment by marcus-j-davies** [comment]
No worries,

If I understand it correctly, we are replacing `\r` with... well... nothing "" so leaving `\n` in tact?
so any`\r\n` will change to `\n`

Re:
>Please remove \r and \t from the descriptions

**Comment by blhoward2** [comment]
> @blhoward2 @AlCalzone ,
> 
> Not sure if this is too late, but #1473 mentions to remove `\r` and / or `\n`
> is it too late to keep them? I can swop these new line characters for `<br />` at render time.
> 
> Some of the metadata's are very "busy" hence the line breaks make it easier on the eye.
> 
> Screenshot 2021-01-24 at 13 14 07

I'm going to redo these in a new PR.

**Comment by marcus-j-davies** [comment]
> I'm going to redo these in a new PR.

I think it will look pretty slick once formatted (replacing `\r\n`with `\n`)

Screenshot 2021-01-24 at 16 29 04

File: packages/config/config/devices/0x0115/zme_06433.json (hunk 9 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0115/zme_06433.json:71]: I've changed this back to 65535, verified in the manual. I also added a few missing parameters.

File: packages/config/config/devices/0x0115/zme_06433.json (hunk 10 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0115/zme_06433.json:141]: Confirmed 10 ms in manual

**Comment by blhoward2** [comment]
@AlCalzone All ready.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**